index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/30e1c676d3.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main>
+      <header>
+        <strong>Welcome</strong>
+        <i class="fa-solid fa-bars"></i>
```

Comment:
- [ ] I would kindly suggest you wrap your `hamburger` icon inside a `nav` tag, The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.